### PR TITLE
Fix total-saved-data endpoint

### LINKED ISSUE #1307: Re implement  getTotalData
[getTotalData](https://github.com/0chain/0chain/blob/staging/code/go/0chain.net/smartcontract/storagesc/handler.go#L795) was replaced with a placeholder when the aLL_BLOBBERS key was removed from the MPT.

This needs to be reworked. We can first try using an aggregate query with blobber.Used, to see if that is quick enough.

### PR DESCRIPTION
**Pull request by Sriep**
## Fixes
Activates the total-saved-data endpoint
## Changes
In every [commit_connection](https://github.com/0chain/0chain/blob/staging/code/go/0chain.net/smartcontract/storagesc/sc.go#L184) transaction, the [WriteMarker.Size](https://github.com/0chain/0chain/blob/staging/code/go/0chain.net/smartcontract/storagesc/models.go#L1355) is used to increment a total data stored field for a blobber.

This endpoint returns the summations of all WriteMarker.Sized fields processed by 0chain. 

## Tests
Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/0chain/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

## Associated PRs (Link as appropriate):
- blobber:
- gosdk: https://github.com/0chain/gosdk/pull/474
- system_test:
- zboxcli: https://github.com/0chain/zboxcli/pull/245
- zwalletcli:
- Other: ...

### CODE REVIEW

File: code/go/0chain.net/smartcontract/storagesc/handler.go [hunk 1]
```diff
@@ -692,14 +693,25 @@ func (srh *StorageRestHandler) getConfig(w http.ResponseWriter, r *http.Request)
...
 func (srh *StorageRestHandler) getTotalData(w http.ResponseWriter, r *http.Request) {
-	common.Respond(w, r, 0, fmt.Errorf("not implemented yet"))
+	edb := srh.GetQueryStateContext().GetEventDB()
+	if edb == nil {
+		common.Respond(w, r, nil, common.NewErrInternal("no db connection"))
+		return
+	}
+	total, err := edb.TotalUsedData()
+	if err != nil {
+		common.Respond(w, r, nil, common.NewErrInternal("getting block "+err.Error()))
```
Comment (peterlimg) [inline code/go/0chain.net/smartcontract/storagesc/handler.go:710]: Return if got error

**Review by peterlimg — CHANGES_REQUESTED** [review]

**Review by peterlimg — APPROVED** [review]
lgtm

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | master  |
| gosdk  | NOT OVERRIDDEN  |
| zboxcli  | staging  |
| zwalletcli  | staging  |
| 0chain  | staging  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | master  |
| 0proxy  | staging  |
| 0search  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [success] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | master  |
| gosdk  | NOT OVERRIDDEN  |
| zboxcli  | staging  |
| zwalletcli  | staging  |
| 0chain  | staging  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | master  |
| 0proxy  | staging  |
| 0search  | staging  |

**Pull request merged**